### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S ./CHANGES.md,./surface/src/3rdparty,./surface/include/pcl/surface/3rdparty,./recognition/include/pcl/recognition/3rdparty -L afile,ang,bu,childs,coo,currect,dosen,frome,gool,hsi,indeces,ihs,indext,isnt,ith,lod,meshe,metre,metres,nd,opps,ot,shs,te,vertexes`

### CODE REVIEW

File: apps/src/face_detection/openni_face_detection.cpp [hunk 1]
```diff
@@ -19,7 +19,7 @@ run(pcl::RFFaceDetectorTrainer& fdrf, bool heat_map = false, bool show_votes = f
   OpenNIFrameSource::OpenNIFrameSource camera;
   OpenNIFrameSource::PointCloudPtr scene_vis;
 
-  pcl::visualization::PCLVisualizer vis("Face dection");
+  pcl::visualization::PCLVisualizer vis("Face detection");
```
Comment (luzpaz) [inline apps/src/face_detection/openni_face_detection.cpp:22]: source change

Comment (mvieth) [reply apps/src/face_detection/openni_face_detection.cpp:22]: Thanks for the PR! Can you explain what you mean exactly with "source change"?

Comment (luzpaz) [reply apps/src/face_detection/openni_face_detection.cpp:22]: This changes real source code (not documentation or comemnts), AFAICT

Comment (mvieth) [reply apps/src/face_detection/openni_face_detection.cpp:22]: Ah, got it, thanks!

**Review by mvieth — APPROVED** [review]

**Pull request merged**